documentation/book/assembly-upgrade-resources.adoc

```diff
@@ -0,0 +1,18 @@
+// This assembly is included in the following assemblies:
+//
+// master.adoc
+
+[id='assembly-upgrade-resources-{context}']
+= {ProductName} resource upgrades
+For this release of {ProductName}, resources that use the API version `{KafkaApiVersionPrev}` must be updated to use `{KafkaApiVersion}`.
+
+The `{KafkaApiVersionPrev}` API version has been deprecated for this release.
```

Comment:
Should it be `from` instead of `for`? And since this will span multiple releases, should we actually name the release here?

```suggestion
The `{KafkaApiVersionPrev}` API version has been deprecated from release 0.12. It will be removed in the future.
```